### PR TITLE
Correct android build tutorial

### PR DESCRIPTION
**Pull request by chuan9**
BUG=XWALK-7393

### CODE REVIEW

**Comment by chuan9** [comment]
@rakuco PTAL, thansk!

**Comment by rakuco** [comment]
This affects other files as well, can you update them too? `git grep target_arch` in `public/contribute` should list all occurrences (some of them are valid though).

**Comment by chuan9** [comment]
@rakuco Yes, replaced all affected string in contribute section, thanks!

File: public/contribute/building_crosswalk/android_build.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ There are two ways to define those flags:
 
 The very least you need to add to your `GYP_DEFINES` is `OS=android`. The
 default build will target 32-bit ARM; if you want to target a different
-architecture, you must also set the `target_arch` variable to `ia32`, `x64` or
```
Comment (rakuco) [inline public/contribute/building_crosswalk/android_build.md]: Like I said, some occurrences _are_ valid. `target_arch` is actually `target_arch` in gyp.

File: public/contribute/building_crosswalk/android_build.md [hunk 2]
```diff
@@ -35,13 +35,13 @@ There are two ways to define those flags:
...
+architecture, you must also set the `target_cpu` variable to `ia32`, `x64` or
 `arm64`. For example, if you are using `chromium.gyp_env`, it should look like
 this to create an Android build for 64-bit Intel processors.
 
 ```
 {
-  'GYP_DEFINES': 'OS=android target_arch=x64',
+  'GYP_DEFINES': 'OS=android target_cpu=x64',
```
Comment (rakuco) [inline public/contribute/building_crosswalk/android_build.md]: Ditto.

File: public/contribute/building_crosswalk/windows_build.md [hunk 3]
```diff
@@ -70,13 +70,13 @@ There are two ways to define those flags:
 These values are read every time you run the `src/xwalk/gyp_xwalk` script, so
 whenever you want to change them remember to run the script afterwards.
 
-To generate a 64-bit build instead of a 32-bit one, you can set `target_arch`
```
Comment (rakuco) [inline public/contribute/building_crosswalk/windows_build.md]: Again, the gyp variable name is correct.

File: public/contribute/building_crosswalk/windows_build.md [hunk 4]
```diff
@@ -70,13 +70,13 @@ There are two ways to define those flags:
...
+To generate a 64-bit build instead of a 32-bit one, you can set `target_cpu`
 to `x64`. For example, if you are using `chromium.gyp_env`, its contents should
 look like this:
 
 ```
 {
-  'GYP_DEFINES': 'target_arch=x64',
+  'GYP_DEFINES': 'target_cpu=x64',
```
Comment (rakuco) [inline public/contribute/building_crosswalk/windows_build.md]: Ditto.

File: public/contribute/building_crosswalk_zh.md [hunk 5]
```diff
@@ -130,7 +130,7 @@
    在Windows下：
 
    ```
-   set GYP_DEFINES=target_arch=x64
+   set GYP_DEFINES=target_cpu=x64
```
Comment (rakuco) [inline public/contribute/building_crosswalk_zh.md]: Same thing in the whole file.

**Comment by chuan9** [comment]
@rakuco restored the target_arch in gyp part. Thanks for review

**Pull request merged**